.gitignore

```diff
@@ -1,5 +1,5 @@
 ## Ignore WebConfig because it contains sensitive values
-Web.config
+Authy2FA/Web.config
```

Comment:
That's really cool @devinrader, I'll use the proposed approach to use the `Web.config` more properly. This will lead us to update the `.gitignore` file and include back the `Web.config`. Thanks!